File: DemoIntent/app/src/main/java/com/example/demointent/MainActivity.java
package com.example.demointent;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);
            return insets;
        });
        Button btnB1 = findViewById(R.id.btnB1);
        Button btnB2 = findViewById(R.id.btnB2);
        Button btnB3 = findViewById(R.id.btnB3);

        btnB1.setOnClickListener(v -> {
            startActivity(new Intent(MainActivity.this, MenuB1Activity.class));
        });
        btnB2.setOnClickListener(v -> {
            startActivity(new Intent(MainActivity.this, MenuB2Activity.class));
        });
        btnB3.setOnClickListener(v -> {
            startActivity(new Intent(MainActivity.this, MenuB3Activity.class));
        });
    }
}

File: Lab01/app/src/main/java/com/example/lab01/MainActivity.kt
package com.example.lab01

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btn1 = findViewById<Button>(R.id.btnBai1)
        val btn2 = findViewById<Button>(R.id.btnBai2)
        val btn5 = findViewById<Button>(R.id.btnBai5)
        val btn7 = findViewById<Button>(R.id.btnBai7)
        val btn7bs = findViewById<Button>(R.id.btnBai7bs)
        val btn8 = findViewById<Button>(R.id.btnBai8)
        val btn8bs = findViewById<Button>(R.id.btnBai8bs)
        val btn9 = findViewById<Button>(R.id.btnBai9)

        btn1.setOnClickListener {
            val intent = Intent(this, B1Activity::class.java)
            startActivity(intent)
        }
        btn2.setOnClickListener {
            val intent = Intent(this, B2Activity::class.java)
            startActivity(intent)
        }
        btn5.setOnClickListener {
            val intent = Intent(this, B5Activity::class.java)
            startActivity(intent)
        }
        btn7.setOnClickListener {
            val intent = Intent(this, B7Activity::class.java)
            startActivity(intent)
        }
        btn7bs.setOnClickListener {
            val intent = Intent(this, B7bsActivity::class.java)
            startActivity(intent)
        }
        btn8.setOnClickListener {
            val intent = Intent(this, B8Activity::class.java)
            startActivity(intent)
        }
        btn8bs.setOnClickListener {
            val intent = Intent(this, B81Activity::class.java)
            startActivity(intent)
        }
        btn9.setOnClickListener {
            val intent = Intent(this, B9Activity::class.java)
            startActivity(intent)
        }
    }
}